recommend/runtimePolicy.go

```diff
@@ -0,0 +1,150 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 Authors of KubeArmor
+
+package recommend
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	opb "github.com/accuknox/auto-policy-discovery/src/protobuf/v1/observability"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+var ms MatchSpec
+var filePaths, processPaths SysRule
+
+// CreateRuntimePolicy function generates runtime policy for service account
+func CreateRuntimePolicy(o *Options, img *ImageInfo) error {
+	ms = MatchSpec{}
+	var label string
+
+	for _, labels := range o.Labels {
+		kvPair := strings.FieldsFunc(labels, labelSplitter)
+		label = strings.TrimPrefix(fmt.Sprintf("%s,%s=%s", label, kvPair[0], kvPair[1]), ",")
+
+	}
+
+	ms, err := summarySearch(label, o.Namespace)
+
+	img.writePolicyFile(ms)
+
+	return err
+
+}
+
+func summarySearch(labels, namespace string) (MatchSpec, error) {
```

Comment:
This function should not rely on global `ms` variable. It should create a `MatchSpec` object locally and return it.